databricks/koalas/base.py

```diff
@@ -180,8 +180,14 @@ def __sub__(self, other):
 
     def __truediv__(self, other):
         def truediv(left, right):
-            return F.when(F.lit(right == 0), F.lit(np.inf).__div__(left)).otherwise(
-                left.__truediv__(right)
+            return F.when(
```

Comment:
@itholic while we're here, can we document what behaviour this logic is in the docstring? Seems it's getting more difficult to read and follow.

Reply:
@HyukjinKwon 
Yeah, I agree that.
I just simplify the code by considering every case.
could you check the simplified code once more when you available?
If you think It's still confused though, I'll add the proper docstring or comments